### PR TITLE
AV-169470 : AKO Operator 1.9.1 Document update

### PR DESCRIPTION
**Pull request by DixitAakash**

### CODE REVIEW

File: ako-operator/README.md [hunk 1]
```diff
@@ -131,14 +131,36 @@ kubectl apply -f config/secrets/secret.yaml
 #### Deploying the AKO Controller
 
 If the AKO operator was installed using helm, a default `AKOConfig` object called `ako-config` is already added and hence, this step is not required for helm based installation.
-**Note**: If the AKO operator was installed manually, then to install the AKO controller, add an `AKOConfig` object to the `avi-system` namespace.
+**Note**: If the AKO operator was installed manually or on Openshift cluster from OperatorHub, then to install the AKO controller, add an `AKOConfig` object to the `avi-system` namespace.
```
Comment (swathinsankaran) [inline ako-operator/README.md:134]: In the above document the name `Openshift Container Platform Web Console` and here it is `OperatorHub`. Is it ok to have both names? Which one is the actual name of web console?

Comment (DixitAakash) [reply ako-operator/README.md:134]: Operator is installed from OperatorHub which is accessible from Openshift Container Platform.

**Comment by DixitAakash** [comment]
build ako

File: ako-operator/README.md (hunk 1 above)
Comment (swathinsankaran) [reply ako-operator/README.md:134]: got it. thanks

**Review by swathinsankaran — APPROVED** [review]

File: ako-operator/README.md (hunk 1 above)
Comment (akshayhavile) [reply ako-operator/README.md:134]: I think we do not support manual installation of AKO operator using `helm`. right?

Comment (DixitAakash) [reply ako-operator/README.md]: Have commented the helm related information.

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Pull request merged**